### ISSUE TITLE
Failed to create history session

### ISSUE THREAD

**Issue by Wilder-wfo** [spyder-ide/spyder#17914]
### What steps will reproduce the problem?

[SpyderKernelApp] ERROR | Failed to create history session in C:\Users\user\.ipython\profile_default\history.sqlite. History will not be saved.
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\IPython\core\history.py", line 543, in __init__
    self.new_session()
  File "<decorator-gen-15>", line 2, in new_session
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\IPython\core\history.py", line 58, in needs_sqlite
    return f(self, *a, **kw)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\IPython\core\history.py", line 569, in new_session
    NULL, "") """, (datetime.datetime.now(),))
sqlite3.OperationalError: attempt to write a readonly database

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\plugins\completion\container.py", line 37, in show_widget
    widget.exec_()
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\plugins\completion\providers\languageserver\widgets\messagebox.py", line 38, in exec_
    self.set_conf('show_lsp_down_warning', not self.is_checked())
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\plugins\completion\api.py", line 1227, in set_conf
    recursive_notification=recursive_notification)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\api\plugins.py", line 1034, in set_conf
    recursive_notification=recursive_notification)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\config\manager.py", line 483, in set
    section, original_option, recursive_notification)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\config\manager.py", line 330, in notify_observers
    self._notify_option(section, tuple_option, value)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\config\manager.py", line 348, in _notify_option
    observer.on_configuration_change(option, section, value)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\api\config\mixins.py", line 227, in on_configuration_change
    method(value)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\plugins\completion\providers\languageserver\provider.py", line 555, in update_configuration
    self.update_lsp_configuration()
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\plugins\completion\providers\languageserver\provider.py", line 629, in update_lsp_configuration
    new_lang_config['configurations'])
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\plugins\completion\providers\languageserver\decorators.py", line 43, in wrapper
    _id = self.send(method, params, kind)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\spyder\plugins\completion\providers\languageserver\client.py", line 470, in send
    self.zmq_out_socket.send_pyobj(msg, flags=zmq.NOBLOCK)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\zmq\sugar\socket.py", line 719, in send_pyobj
    return self.send(msg, flags=flags, **kwargs)
  File "C:\Users\user\AppData\Local\Programs\Spyder\pkgs\zmq\sugar\socket.py", line 505, in send
    return super(Socket, self).send(data, flags=flags, copy=copy, track=track)
  File "zmq\backend\cython\socket.pyx", line 718, in zmq.backend.cython.socket.Socket.send
  File "zmq\backend\cython\socket.pyx", line 759, in zmq.backend.cython.socket.Socket.send
  File "zmq\backend\cython\socket.pyx", line 135, in zmq.backend.cython.socket._check_closed
zmq.error.ZMQError: not a socket
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.7.9
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree                  :  None (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.4 (OK)
setuptools >=39.0.0           :  54.2.0 (OK)
sphinx >=0.6.6                :  3.5.3 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.1 (NOK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog                      :  1.0.2 (OK)
zmq >=17                      :  22.0.3 (OK)

# Optional:
cython >=0.21                 :  0.29.22 (OK)
matplotlib >=2.0.0            :  3.4.1 (OK)
numpy >=1.7                   :  1.19.3 (OK)
pandas >=1.1.1                :  1.2.3 (OK)
scipy >=0.17.0                :  1.6.2 (OK)
sympy >=0.7.3                 :  1.7.1 (OK)
```

**Comment by dalthviz** [comment]
Hi @Wilder-wfo thank you for the feedback! Seems like you are using an old standalone Spyder version. Could you update by uninstalling your current version and installing the latest available (Spyder 5.3.0) ? https://github.com/spyder-ide/spyder/releases/tag/v5.3.0

Let us know if that helps!

**Comment by dalthviz** [comment]
Closing due to lack of response. Also, just in case, the latest Spyder version available at the moment is 5.3.1. The latest release can be found always at: https://github.com/spyder-ide/spyder/releases/latest
